APIManager.js

```diff
@@ -0,0 +1,81 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+
+    constructor() {
+        this.data = {
+            user: {},
```

Comment:
you do not need to add keys to `this.data` If you remove them it should still work because you are initializing them in the methods below.